api/collector/v1/collector.proto

```diff
@@ -10,19 +10,47 @@ option go_package = "github.com/grafana/alloy-remote-config/api/gen/proto/go/col
 message GetConfigRequest {
   string id = 1;
   map<string, string> attributes = 2;
+  bool is_collector = 3;
```

Comment:
Do we expect GetConfig to have any side effects for _some_ calls or not (after introducing the new endpoints)?

Reply:
Yes, when `is_collector == true` there could be side effects (and no side effects when it is `false`). Are you thinking we should separate this into two endpoints, instead?

Reply:
I mean, do we have a usecase in mind for any side effects?